integration/solana/debug_buffer.sol

```diff
@@ -0,0 +1,154 @@
+import 'solana';
+
+@program_id("PriZbiyLjDtLWb2AcExcE7pxCkesSJzb7Z1x52tUYXt")
```

Comment:
You can remove this line and remove the file `Printer.key`. The file will be generated for each run, no need to check the file in.